docs/user-guide/compute-resources.md

```diff
@@ -269,6 +269,89 @@ LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-0
 
 We can see that this container was terminated because `reason:OOM Killed`, where *OOM* stands for Out Of Memory.
 
+## Alpha Features
+
+### Opaque Integer Resources (Alpha Feature)
+
+Opaque integer resources were introduced as an alpha feature in Kubernetes
```

Comment:
Stick with present tense; no past or future.

"Kubernetes version 1.5 introduces Opaque integer resources. Opaque integer resources allow cluster operators to advertise  new node-level resources that would be otherwise unknown to the system."

Move the last sentence: "So far only resource accounting is implemented" into a note about the Alpha state underneath the first paragraph, like so:

"**Note:** Opaque integer resources are Alpha in Kubernetes version 1.5. Only resource accounting is implemented; node-level isolation is still under active development."